server.go

```diff
@@ -584,31 +578,32 @@ func (s *Server) HandleMetricPacket(packet []byte) error {
 // HandleTracePacket accepts an incoming packet as bytes and sends it to the
 // appropriate worker.
 func (s *Server) HandleTracePacket(packet []byte) {
-	// TODO: don't use statsd for this in favor of an in-memory counter
-	s.Statsd.Incr("ssf.received_total", nil, .1)
+	samples := []*ssf.SSFSample{ssf.Count("veneur.ssf.received_total", 1, nil)}
```

Comment:
This creates a different concern. Multiple `Counts` of the same metric should likely be aggregated, no?

Reply:
That's a good question: Should we do this in the client too or is it enough to do it in the server? Since they're relatively small, I think we could just let the server deal & introduce one fewer place for things to go wrong...